semgrep-core/parsing/Parse_csharp_tree_sitter.ml

```diff
@@ -62,6 +62,18 @@ let arg_to_expr (a : argument) =
   | ArgKwd (_, e) -> e (* TODO maybe ArgKwd is also impossible here *)
   | _ -> raise Impossible
 
+let var_def_stmt (decls : (entity * variable_definition) list) (attrs : attribute list) =
+  let stmts = List.map (fun (ent, def) ->
+    let ent = {
```

Comment:
in OCaml you can write let ent = { ent with attrs = ent.attrs @ attrs }
and the other ent field will be copied.